polaris-react/src/components/Tabs/tests/Tabs.test.tsx

```diff
@@ -206,10 +204,7 @@ describe('<Tabs />', () => {
 
       tabsWithContent.forEach((tab, index) => {
         expect(wrapper.find('ul')!.findAll(Tab)[index]).toContainReactComponent(
-          Text,
-          {
-            children: tab.content,
-          },
+          tab!.content as unknown as ComponentType<any>,
```

Comment:
Fixed up this test but had to cast as unkown first. Tried a few different things but this was the only thing that didn't complain for me